_config.yml

```diff
@@ -34,7 +34,7 @@ ghrepo: "facebook/Site-Templates"
 # Use these color settings to determine your colour scheme for the site.
 color:
   # primary should be a vivid color that reflects the project's brand
-  primary: "#3b5998"
+  primary: "#0f2647"
```

Comment:
Ugh -- can't make a comment on line 21 :/

Anyway, for `url`, it should be:

`url: http://facebook.github.io/`

and on line 15, you should add a `baseurl` that is:

`baseurl: mysql-5.6`